### PR TITLE
Fix syntax error in job pre-condition

### PR DESCRIPTION
**Pull request by parfeon**
build(workflow): fix syntax error in job pre-condition

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -3,50 +3,45 @@ name: Tests
...
+    shell: bash
 
 jobs:
   tests:
     name: Integration and Unit tests
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python: [3.7.13, 3.8.13, 3.9.13, 3.10-dev]
```
Comment (seba-aln) [inline .github/workflows/run-tests.yml:20]: I would check if there is GA release of 3.10 AND 3.11 to cover all currently supported versions. At a time of adding this gh had no ready package for GA 3.10  :(

**Review by seba-aln — APPROVED** [review]
Apart of py version i have no problems with this PR. Good Job
snake-opens

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (parfeon) [reply .github/workflows/run-tests.yml:20]: @seba-aln but we can add it afterwards :)
Right now, I need those changes to complete setup along with Terraform, which will require new set of completed jobs.

Comment (parfeon) [reply .github/workflows/run-tests.yml:20]: @seba-aln it looks like last known is 3.12 according to their [manifest](https://github.com/actions/python-versions/blob/main/versions-manifest.json) file.

Comment (kleewho) [reply .github/workflows/run-tests.yml:20]: Yeah. I guess we could change it to: `[3.7, 3.8, 3.9, 3.10, 3.11]`. I don't think we care much about exact patch version. 3.12 is not yet released so what we could do is to have `3.12-dev`

**Pull request merged**